src/coreclr/gc/gc.cpp

```diff
@@ -9496,7 +9444,7 @@ void gc_heap::append_to_mark_list (uint8_t **start, uint8_t **end)
     size_t slots_to_copy = min(slots_needed, slots_available);
     memcpy(mark_list_index, start, slots_to_copy*sizeof(*start));
     mark_list_index += slots_to_copy;
-//    printf("heap %d: appended %Id slots to mark_list\n", heap_number, slots_to_copy);
+    dprintf (2, ("h%d: appended %Id slots to mark_list\n", heap_number, slots_to_copy));
```

Comment:
You may want to use a higher level here - there may be many of these per GC.

Reply:
will do.